protocol/communities/manager.go

```diff
@@ -1526,6 +1596,154 @@ func (m *Manager) IsSeedingHistoryArchiveTorrent(communityID types.HexBytes) boo
 	return ok && torrent.Seeding()
 }
 
+func (m *Manager) DownloadHistoryArchivesByMagnetlink(communityID types.HexBytes, magnetlink string) ([]string, error) {
+
+	id := communityID.String()
+	ml, _ := metainfo.ParseMagnetUri(magnetlink)
```

Comment:
Is it possible that ParseMagnetUri fails if the magnet link somehow is incorrect?
The reason for the question is that it seems the a possible error here is being ignored.

Reply:
Yeap good point!